beta/src/pages/learn/updating-objects-in-state.md

```diff
@@ -118,101 +125,108 @@ onPointerMove={e => {
 
 With `setPosition`, you're telling React:
 
-* Replace `position` with this new object
-* And render this component again
+- Replace `position` with this new object
```

Comment:
Seems like all `*` got replaced with `-` as well 😕.

Reply:
Yes, I think my formeter is applied. I have restored all changes.

thanks!